### DISCUSSION TITLE
svelte devtools / dev mode

### DISCUSSION THREAD

**Discussion by davidsielert** [templatus/templatus-inertia#1328] [Q&A] [answered]
Is there any possibility to use sevelte devtools with this setup? I had tried it but it shows the following
![image](https://github.com/templatus/templatus-inertia/assets/456592/1fe85b14-6ddb-48f6-a138-c1c6c6acaeda)

**Comment by davidsielert** [comment, marked as answer]
Sorry, I wasn't familiar with this toolset. turns out it does work you need to refresh with the window open (as it says in the picture)

**Comment by ledermann** [comment]
Glad you figured it out :)
